### PR TITLE
N°7184 - Update czech translations thanks to @tacsaby

### PR DESCRIPTION
**Pull request by Stetinac**
## Base information
| Question                                                      | Answer 
|---------------------------------------------------------------|--------
| Related to a SourceForge thead / Another PR / Combodo ticket? | No
| Type of change?                                               |Translations

## Checklist before requesting a review

- [X] I have performed a self-review of my code
- [X] I have tested all changes I made on an iTop instance
- [X] Would a unit test be relevant and have I added it? No
- [X] Is the PR clear and detailled enough so anyone can understand digging in the code?

### CODE REVIEW

File: datamodels/2.x/itop-portal-base/dictionaries/cs.dict.itop-portal-base.php [hunk 1]
```diff
@@ -135,7 +135,7 @@
 	'Brick:Portal:Object:Form:Message:ObjectSaved' => '%1$s uloženo',
 	'Brick:Portal:Object:Search:Regular:Title' => 'Vybrat %1$s (%2$s)',
 	'Brick:Portal:Object:Search:Hierarchy:Title' => 'Vybrat %1$s (%2$s)',
-	'Brick:Portal:Object:Copy:TextToCopy' => '%2$s',
+	'Brick:Portal:Object:Copy:TextToCopy' => '%1$s: %2$s~~',
```
Comment (Molkobain) [inline datamodels/2.x/itop-portal-base/dictionaries/cs.dict.itop-portal-base.php]:
```suggestion
	'Brick:Portal:Object:Copy:TextToCopy' => '%1$s: %2$s',
```

File: dictionaries/cs.dictionary.itop.ui.php [hunk 2]
```diff
@@ -374,7 +374,7 @@
 	'Expression:Unit:Short:DAY' => 'd',
 	'Expression:Unit:Short:WEEK' => 'w',
 	'Expression:Unit:Short:MONTH' => 'm',
-	'Expression:Unit:Short:YEAR' => 'y',
+	'Expression:Unit:Short:YEAR' => 'y~',
```
Comment (Molkobain) [inline dictionaries/cs.dictionary.itop.ui.php]:
```suggestion
	'Expression:Unit:Short:YEAR' => 'y',
```

File: dictionaries/cs.dictionary.itop.ui.php [hunk 3]
```diff
@@ -449,24 +449,24 @@
...
 	'UI:Button:Ok' => 'Ok',
 	'UI:Button:Save' => 'Uložit',
-	'UI:Button:SaveAnd' => 'Save and %1$s~~',
+	'UI:Button:SaveAnd' => 'Uložit a %1$s',
 	'UI:Button:Cancel' => 'Zrušit',
 	'UI:Button:Close' => 'Zavřít',
 	'UI:Button:Apply' => 'Použít',
 	'UI:Button:Send' => 'Odeslat',
-	'UI:Button:SendAnd' => 'Odeslat a %1$s',
+	'UI:Button:SendAnd' => 'Odeslat a %1$s~~',
```
Comment (Molkobain) [inline dictionaries/cs.dictionary.itop.ui.php]:
```suggestion
	'UI:Button:SendAnd' => 'Odeslat a %1$s',
```

File: dictionaries/cs.dictionary.itop.ui.php [hunk 4]
```diff
@@ -565,8 +565,8 @@
 	'UI:Menu:Actions' => 'Akce',
 	'UI:Menu:OtherActions' => 'Další akce',
 	'UI:Menu:Transitions' => 'Transitions~~',
-	'UI:Menu:OtherTransitions' => 'Other transitions~~',
-	'UI:Menu:View' => 'Zobrazit objekt',
+	'UI:Menu:OtherTransitions' => 'Other Transitions~~',
```
Comment (Molkobain) [inline dictionaries/cs.dictionary.itop.ui.php]: Is it intended?

File: dictionaries/cs.dictionary.itop.ui.php [hunk 5]
```diff
@@ -1207,15 +1207,15 @@
...
+	'UI:ArchiveMode:Banner' => 'Archivní mó',
+	'UI:ArchiveMode:Banner+' => 'Archivované objekty jsou nyní vyditelné, ale je zakázána jejich úprava',
 	'UI:FavoriteOrganizations' => 'Oblíbené organizace',
 	'UI:FavoriteOrganizations+' => 'Zaškrtněte, které organizace chcete vidět v rozbalovacím menu pro rychlý přístup. Mějte na paměti, že toto není bezpečnostní opatření. Objekty všech organizací jsou pořád viditelné a přístupné vybráním "Všechny organizace" z rozbalovacího menu.',
-	'UI:FavoriteLanguage' => 'Jazyk uživatelského rozhraní~~',
+	'UI:FavoriteLanguage' => 'Jazyk uživatelského rozhraní',
 	'UI:Favorites:SelectYourLanguage' => 'Preferovaný jazyk:',
 	'UI:FavoriteOtherSettings' => 'Další nastavení',
-	'UI:Favorites:Default_X_ItemsPerPage' => 'Výchozí délka seznamů: %1$s položek na stránku',
+	'UI:Favorites:Default_X_ItemsPerPage' => 'Výchozí délka seznamů: %1$s položek na stránku~~',
```
Comment (Molkobain) [inline dictionaries/cs.dictionary.itop.ui.php]:
```suggestion
	'UI:Favorites:Default_X_ItemsPerPage' => 'Výchozí délka seznamů: %1$s položek na stránku',
```

Comment (Stetinac) [reply dictionaries/cs.dictionary.itop.ui.php]: This is an error from our internal merging scripts :-(

File: dictionaries/cs.dictionary.itop.ui.php (hunk 4 above)
Comment (Stetinac) [reply dictionaries/cs.dictionary.itop.ui.php]: This is an error from our internal merging scripts :-(

File: dictionaries/cs.dictionary.itop.ui.php (hunk 3 above)
Comment (Stetinac) [reply dictionaries/cs.dictionary.itop.ui.php]: This is an error from our internal merging scripts :-(

File: dictionaries/cs.dictionary.itop.ui.php (hunk 2 above)
Comment (Stetinac) [reply dictionaries/cs.dictionary.itop.ui.php]: This is an error from our internal merging scripts :-(

File: datamodels/2.x/itop-portal-base/dictionaries/cs.dict.itop-portal-base.php (hunk 1 above)
Comment (Stetinac) [reply datamodels/2.x/itop-portal-base/dictionaries/cs.dict.itop-portal-base.php]: This is an error from our internal merging scripts :-(

File: dictionaries/cs.dictionary.itop.ui.php (hunk 4 above)
Comment (Molkobain) [inline dictionaries/cs.dictionary.itop.ui.php]:
```suggestion
	'UI:Menu:OtherTransitions' => 'Other transitions~~',
```

File: dictionaries/cs.dictionary.itop.core.php [hunk 6]
```diff
@@ -483,11 +483,11 @@
 	'Class:ActionEmail/Attribute:status/Value:disabled+' => 'The email notification will not be sent~~',
 	'Class:ActionEmail/Attribute:test_recipient' => 'Adresát pro test',
 	'Class:ActionEmail/Attribute:test_recipient+' => 'Cílová adresa pro případ, kdy je stav nastaven na "Testování"',
-	'Class:ActionEmail/Attribute:from' => 'Odesílatel',
+	'Class:ActionEmail/Attribute:from' => 'Odesílatel~~',
```
Comment (Molkobain) [inline dictionaries/cs.dictionary.itop.core.php]:
```suggestion
	'Class:ActionEmail/Attribute:from' => 'Odesílatel',
```

File: dictionaries/cs.dictionary.itop.core.php [hunk 7]
```diff
@@ -483,11 +483,11 @@
...
 	'Class:ActionEmail/Attribute:from+' => '',
 	'Class:ActionEmail/Attribute:from_label' => 'From (label)~~',
 	'Class:ActionEmail/Attribute:from_label+' => 'Sender display name will be sent into the email header~~',
-	'Class:ActionEmail/Attribute:reply_to' => 'Odpověď na',
+	'Class:ActionEmail/Attribute:reply_to' => 'Odpověď na~~',
```
Comment (Molkobain) [inline dictionaries/cs.dictionary.itop.core.php]:
```suggestion
	'Class:ActionEmail/Attribute:reply_to' => 'Odpověď na',
```

**Review by Molkobain — APPROVED** [review]

File: dictionaries/cs.dictionary.itop.core.php [hunk 8]
```diff
@@ -511,20 +511,20 @@
...
+	'ActionEmail:trigger' => 'Spouštěče',
 	'ActionEmail:recipients' => 'Kontakty',
 	'ActionEmail:preview_tab' => 'Náhled',
-	'ActionEmail:preview_tab+' => 'Preview of the eMail template~~',
-	'ActionEmail:preview_warning' => 'The actual eMail may look different in the eMail client than this preview in your browser.~~',
-	'ActionEmail:preview_more_info' => 'For more information about the CSS features supported by the different eMail clients, refer to %1$s~~',
-	'ActionEmail:content_placeholder_missing' => 'The placeholder "%1$s" was not found in the HTML template. The content of the field "%2$s" will not be included in the generated emails.~~',
+	'ActionEmail:preview_tab+' => 'Náhled emailové šablony',
+	'ActionEmail:preview_warning' => 'Finální podoba emailu může vypadat v emailovém klientu jinak, než tento aktuální náhled v prohlížeci',
+	'ActionEmail:preview_more_info' => 'Další informace ohledně podpory CSS v emailových klientech, se dozvíte zde %1$s~~',
```
Comment (Molkobain) [inline dictionaries/cs.dictionary.itop.core.php]:
```suggestion
	'ActionEmail:preview_more_info' => 'Další informace ohledně podpory CSS v emailových klientech, se dozvíte zde %1$s',
```

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
Hello Pavel,

I'm sorry but I wasn't able to merge your PR correctly as it was not on a dedicated branch, but on the `develop` branch. Please mind to do a dedicated branch for each PR next time :)

In the meantime, I retrieved your commits and merged them manually with [962eb08](https://github.com/Combodo/iTop/commit/962eb08e40edebbc99c60f0100b46773a9a80037) :)

**Pull request closed without merging**